### PR TITLE
fix(components): Change `Table` borders from box-shadow to pseudo element

### PR DESCRIPTION
**Pull request by eddysims**
## Motivations

The `Table` was using `box-shadow` as borders due to the fact that `<table>` can not have a `border-radius` applied to it. While this looks fine when a `Table` is by its self. It looks a little bonkers when paired with something like a `Card`.

## Changes

Change the `Table` borders from `box-shadow` to a `pseudo` element.

### Added

- 

### Changed

- Change the `Table` borders from `box-shadow` to a `pseudo` element.

### Deprecated

- 

### Removed

- 

### Fixed

- 

### Security

- 

## Testing

---

[In Atlantis we use Github's built in pull request reviews](https://help.github.com/en/articles/about-pull-request-reviews).

Random photo of Atlantis

### CODE REVIEW

File: packages/components/src/Table/Table.css [hunk 1]
```diff
@@ -2,13 +2,28 @@
...
+
+.table::after {
+  content: "";
+  position: absolute;
+  top: 0;
+  left: 0;
+  z-index: 1;
+  width: calc(100% - 1px);
+  height: 100%;
+  border: solid 1px var(--border-color);
```
Comment (chris-at-jobber) [inline packages/components/src/Table/Table.css:18]: Can we do `solid var(--border-base) var(--border-color)`?

**Comment by chris-at-jobber** [comment]
@eddysims left a quick comment, I can take a more thorough look and do some testing tomorrow!

File: packages/components/src/Table/Table.css (hunk 1 above)
Comment (chris-at-jobber) [reply packages/components/src/Table/Table.css:18]: Also if the 1px on line 16 is also based on desired border width, this would apply there as well

Comment (eddysims) [reply packages/components/src/Table/Table.css]: changed it to css custom properties

**Comment by chris-at-jobber** [comment]
Seeing a lil' baby gap here: 
image

**Comment by eddysims** [comment]
> Seeing a lil' baby gap here:
> image

image

@chris-at-jobber Fixed!

**Review by chris-at-jobber — APPROVED** [review]
Looks good to me!

**Pull request merged**